pkgs/build-support/agda/default.nix

```diff
@@ -76,7 +77,7 @@ let
           find -not \( -path ${everythingFile} -or -path ${lib.interfaceFile everythingFile} \) -and \( ${concatMapStringsSep " -or " (p: "-name '*.${p}'") (extensions ++ extraExtensions)} \) -exec cp -p --parents -t "$out" {} +
           runHook postInstall
         '';
-      };
+      } // (if broken then { hydraPlatforms = lib.platforms.none; } else {});
```

Comment:
`broken` and `hydraPlatforms` both need to go into the `meta` set, so this change will probably have no effect at all.

Reply:
Better?